packages/libs/react-ui/src/components/TextArea/TextArea.css.ts

```diff
@@ -0,0 +1,107 @@
+import { statusColor } from '../InputWrapper/InputWrapper.css';
+
+import { sprinkles } from '@theme/sprinkles.css';
+import { darkThemeClass, vars } from '@theme/vars.css';
+import { fallbackVar, style } from '@vanilla-extract/css';
+
+export const containerClass = style([
+  sprinkles({
+    alignItems: 'stretch',
+    borderRadius: '$sm',
+    display: 'flex',
+    color: '$foreground',
+    overflow: 'hidden',
+    lineHeight: '$lg',
+    bg: {
+      lightMode: '$white',
+      darkMode: '$gray100',
+    },
+  }),
+  {
+    position: 'relative',
+    borderBottom: `1px solid ${fallbackVar(statusColor, vars.colors.$gray30)}`,
+    selectors: {
+      [`${darkThemeClass} &`]: {
+        borderBottom: `1px solid ${fallbackVar(
+          statusColor,
+          vars.colors.$gray60,
+        )}`,
+      },
+      '.inputGroup &': {
+        borderRadius: 0,
+      },
+      '.inputGroup &:first-child': {
+        borderTopRightRadius: vars.radii.$sm,
+        borderTopLeftRadius: vars.radii.$sm,
+      },
+      '.inputGroup &:last-child': {
+        borderBottomRightRadius: vars.radii.$sm,
+        borderBottomLeftRadius: vars.radii.$sm,
+      },
+    },
+  },
+]);
+
+export const disabledClass = style([
+  sprinkles({
+    pointerEvents: 'none',
+    bg: {
+      darkMode: '$gray60',
+      lightMode: '$gray20',
+    },
+  }),
+  {
+    opacity: 0.4,
+    selectors: {
+      '.inputGroup &': {
+        opacity: 1,
+      },
+      [`${darkThemeClass} &`]: {
+        backgroundColor: vars.colors.$gray60,
+      },
+    },
+  },
+]);
+
+export const textAreaContainerClass = style([
+  sprinkles({
+    position: 'relative',
+    alignItems: 'center',
+    display: 'flex',
+    flexGrow: 1,
+    gap: '$2',
+    lineHeight: '$lg',
+    paddingX: '$4',
+  }),
+]);
+
+export const textAreaClass = style([
+  sprinkles({
+    alignItems: 'center',
+    background: 'none',
+    border: 'none',
+    color: '$foreground',
+    outline: 'none',
+    flexGrow: 1,
+    paddingY: '$2',
+    minHeight: '$20',
+  }),
+  {
+    resize: 'none',
+    '::placeholder': {
+      color: vars.colors.$gray40,
+    },
+    [`${darkThemeClass} &::placeholder`]: {
+      color: vars.colors.$gray50,
+    },
+  },
+]);
+
+export const outlinedClass = style([
+  sprinkles({
+    borderRadius: '$sm',
+  }),
+  {
+    border: `1px solid ${vars.colors.$neutral3}`,
+  },
+]);
```

Comment:
I believe this came from the input component, but this should actually be part of a variant since it is a style. Also this would align better with other components. I've added it as a task on the asana board to update in the future https://app.asana.com/0/1205553603119741/1205687963085321/f

Reply:
this is actually outdated as part of the new story im doing, which will make it always outlined and the status colors will apply to this border as well